service/sysconfig/observiq-otel-collector

```diff
@@ -0,0 +1,18 @@
+# Configuration file for the observiq-otel-collector service.
+
+# The server keys are automatically generated if they ommited
+# to change the automatic creation uncomment the approprite 
+# line.
```

Comment:
Is this a hold over from something else?

Reply:
Yes, the file I copied it from. Fixing now.